### DISCUSSION TITLE
Dell G16 7630 Fan Control?

### DISCUSSION THREAD

**Discussion by jst200** [T-Troll/alienfx-tools#365] [General]
Hi,

When running AlienFX Fan Control on my Dell G16 7630 it shows a constant 60C for both the SSDs, whatever their actual temperature. Please see screen shot below (please note, it's pure coincidence that the CrystalDisk disp;lay of the C: drive in the Icon Bar is showing 60C at the time of the screenshot ;-) )

![Screenshot 2023-08-08 165206](https://github.com/T-Troll/alienfx-tools/assets/12275611/1f48b480-b3d5-407f-905e-7ca4f33eb82f)

I know the G16 is not on your supported list, but I thought I read a note to the effect that it should (might :-) ) be compatible with any G series laptops. So was hoping it might work anyway.

The problem I have is that the "as supplied" C: drive SSD has a tendency to get hot very quickly. The D: drive Crucial SSD I added is better, as I added a heatsink, but it too can get hot if pushed. Currently I monitor the SSD temps using CrystalDisk and when the SSDs start to get hot I use AWCC V6 to manually boost the mode to "Performance" or "G mode". This spins up the CPU/GPU fans and helps cool the disks.

I was hoping that I could use AlienFX to automate the fan speed based on the SSD temp.

I am more than happy to help in anyway to introduce support for the Dell G16 7630.

Any thoughts/comment welcome.

Thanks, Jim :-)

**Comment by T-Troll** [comment]
Well... As far as i see, your notebook works too (thanks to common interface). Is it AMD (it only have 2 common Alienware sensors)?

Let me explain you some.
SSD temperatures come from Windows WMI (SMART) values reported by drive in itself. So, seems like, they don't report it correctly.
Just ignore it in this case.
It's quite common in case you have RAID0 array.
